src/components/icons/BaseSvg.svelte

```diff
@@ -3,12 +3,13 @@
 	export let title: string | undefined = undefined;
 
 	$: labelled = $$props['aria-label'] || $$props['aria-labelledby'] || title;
+	$: viewBox = `0 0 ${size} ${size}`;
```

Comment:
The `viewBox` shouldn't equal `size`. For SVG, viewBox is like a min-max coordinate which each element in the SVG will reference.  So viewBox is fixed, and it is the size of the original size of SVG (that you export from Figma)

For example, `d` property in the path element of the social icon won't have any value less than 0 or more than 32, since the original size is 32, so the `viewBox` must be 0 0 32 32. While other icons I created will be between 0-16 because the original size is 16, so the `viewBox` must be 0 0 16 16.

If the `size` is not the same as `viewBox`, the browser will just scale that for us. So I suggest you either not change anything in this file, or turn `viewBox` into a prop with a default value of `"0 0 16 16"`.

Reply:
Oh, I think I just forgot to remove it 🥺